### PR TITLE
Ayr 712/filter with any character

### PR DESCRIPTION
**Pull request by pgandhizaizi**
### After

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/main/db/queries.py [hunk 1]
```diff
@@ -256,14 +256,14 @@ def _build_series_view_query(series_id, filters, sorting_orders):
 def _build_browse_filters(query, sub_query, filters):
     transferring_body = filters.get("transferring_body")
     if transferring_body:
-        filter_value = str(f"{transferring_body}%").lower()
+        filter_value = str(f"%{transferring_body}%").lower()
```
Comment (anthonyhashemi) [inline app/main/db/queries.py:259]: so I assume %STRING% is the format for a CONTAINS string filter in sqlalchmey

Comment (pgandhizaizi) [reply app/main/db/queries.py:259]: yes, it refers to like operator of database

Comment (pgandhizaizi) [reply app/main/db/queries.py:259]: if you want to match characters that are contained in a given string, you surround the string with percentage signs (%) on both sides of the filter value string. i.e. %t% will get results for test, star, dest, into and so on

Comment (anthonyhashemi) [reply app/main/db/queries.py:259]: perfect, thanks for explaining 👌

File: e2e_tests/test_browse.py [hunk 2]
```diff
@@ -74,6 +74,46 @@ def test_browse_sort_and_filter_functionality(
...
+            "Transferring body",
+            "Series",
+            "Last record transferred",
+            "Records held",
+            "Consignments within series",
+        ]
+
+        assert rows.count() == 2
+
+        if cols.count() > 0:
```
Comment (anthonyhashemi) [inline e2e_tests/test_browse.py]: can we remove these conditions like in the last PR?

Comment (pgandhizaizi) [reply e2e_tests/test_browse.py]: condition removed as requested

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**